### PR TITLE
remove unused field and complete fireLaser method

### PR DESCRIPTION
**Pull request by MateSzoke**

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/GT4500.java [hunk 1]
```diff
@@ -21,8 +16,10 @@ public GT4500() {
   }
 
   public boolean fireLaser(FiringMode firingMode) {
-    // TODO not implemented yet
-    return false;
+   	if(1 < 2)
+   	 return false;
```
Comment (matesebi) [inline src/main/java/hu/bme/mit/spaceship/GT4500.java]: this implementation does not meet the specifications requirements

**Review by matesebi — CHANGES_REQUESTED** [review]

File: src/main/java/hu/bme/mit/spaceship/GT4500.java (hunk 1 above)
Comment (MateSzoke) [reply src/main/java/hu/bme/mit/spaceship/GT4500.java]: :(

Comment (matesebi) [reply src/main/java/hu/bme/mit/spaceship/GT4500.java]: just do it

**Review by matesebi — APPROVED** [review]
nice

**Pull request merged**